assets/js/aplus.js

```diff
@@ -463,9 +464,21 @@ $(function() {
             this.element.hide();
             var self = this;
             this.selection = this.widget.find(this.settings.selection_selector);
+            this.api_url = self.element.attr("data-users-api-url")
             this.selection_li = this.selection.find("li").remove();
             this.element.find("option:selected").each(function(index) {
-                self.addSelection($(this).attr("value"), $(this).text());
+              $.ajax({
+                url: self.api_url,
+                data: {
+                  "search": $(this).attr("value"),
```

Comment:
This makes a GET request for each existing assistant in the course. It would be more efficient if they could be combined to single GET request that retrieves the user data for the given user IDs. 
If we keep this approach that makes GET requests for each user, then it must use the user detail API URL instead of making a useless seach in the API. (User details is something like `/api/v2/users/ID`).

Reply:
For now there is separate GET request for each assistant. It will probably take 50-200ms to load per assistant, so even in the larger courses the delay shouldn't be unbearable, since the 'Edit course details' page is not used very often.